src/host/testbench.c

```diff
@@ -228,6 +231,16 @@ int main(int argc, char **argv)
 			parse_libraries(optarg, lib_table);
 			break;
 
+		/* input sample rate */
+		case 'x':
```

Comment:
x & y ? what about r & R for rates ? easier to remember.

Reply:
Script src_testh.sh in SOFT/test uses -x and -y so it needs to be changed if these change. Maybe the getopt that's used is not capable but I'd prefer the long "--style" since we have a lot of command line switches already.